GrayBMP.cs

```diff
@@ -126,6 +125,39 @@ public void Dirty ()
       End ();
    }
 
+   /// <summary>
+   /// Draws a thick line between the two given end-points (with given size & given shade of gray)
+   /// </summary>
+
+   public void DrawThickLine (int x1, int y1, int x2, int y2, int width, int color) {
+      Begin (); 
+      Check (x1, y1); Check (x2, y2); Dirty (x1, y1, x2, y2);
+
+      const double OneRadian = PI / 180;
+      double offset = width * 0.5;
+      var BaseAngle = Atan2 (y2 - y1, x2 - x1) / OneRadian;

```

Comment:
Variable name should start with small letter.

Reply:
Try working with radian instead of converting into degree and converting back to radian.
Hint: 60 in degree = Math.PI / 3 in radian.